scripts/update/jellyfin.sh

```diff
@@ -9,79 +9,79 @@ if [[ -f /install/.jellyin.lock ]]; then
     username="$(_get_master_username)"
     #
     # disable the service to make usre the mosrt up to date and correct service file is installed via apt.
-    systemctl -q stop "jellyfin.service"
+    systemctl -q stop jellyfin.service
```

Comment:
this will still stop JF every time `box update` is ran, regardless of whether or not it has been migrated